Blizzard.IO.RabbitMQ.Tests/Rpc/NetqRabbitRpcServerTests.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Blizzard.IO.RabbitMQ.Rpc;
+using EasyNetQ;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Tests.Rpc
+{
+    class RequestStub
```

Comment:
Create a stub folder and move the two classes there

Reply:
f